docs/.gitignore

```diff
@@ -2,3 +2,6 @@ build/
 source/examples/cpp/*
 source/examples/dpc/*
 docbuild-log.txt
+__pycache__
+doxygen/oneapi/doxygen/xml
+.vs*
```

Comment:
Isn't .vs defined by gitignore in the repo root, if it's not, I'd recommend to put it there

Reply:
done